### PR TITLE
Cleanup (following #17098) for HLT ConfDB parsing

### PR DESCRIPTION
**Pull request by Martin-Grunewald**
Cleanup (following #17098) for HLT ConfDB parsing.
PR based on CMSSW_9_0_X_2017-02-06-2300.

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
type bugfix

**Comment by Martin-Grunewald** [comment]
please abort

File: RecoTracker/TkSeedGenerator/python/SeedGeneratorFromProtoTracksEDProducer_cff.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 pixelTTRHBuilderWithoutAngle = copy.deepcopy(ttrhbwr)
 from RecoTracker.TkSeedingLayers.TTRHBuilderWithoutAngle4PixelTriplets_cfi import *
 from RecoTracker.TkSeedingLayers.PixelLayerTriplets_cfi import *
-from RecoPixelVertexing.PixelTrackFitting.PixelTracks_cfi import *
+from RecoPixelVertexing.PixelTrackFitting.pixelTracksDefault_cfi import *
```
Comment (makortel) [inline RecoTracker/TkSeedGenerator/python/SeedGeneratorFromProtoTracksEDProducer_cff.py]: Whoops, something I missed in #17170. More proper solution would be to add `pixelTracks_cfi.py`, move
```
pixelTracks = _pixelTracksDefault.clone()
_SeedingHitSets = dict(SeedingHitSets = "pixelTracksHitQuadruplets")
trackingPhase1PU70.toModify(pixelTracks, **_SeedingHitSets)
trackingPhase2PU140.toModify(pixelTracks, **_SeedingHitSets)
```
from [RecoPixelVertexing/PixelTrackFitting/python/PixelTracks_cff.py](https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/RecoPixelVertexing/PixelTrackFitting/python/PixelTracks_cff.py#L72-L76) there, and fix all imports.

@Martin-Grunewald Depending on your preference, I can do it now, after this PR, or leave it to you.

Comment (Martin-Grunewald) [inline RecoTracker/TkSeedGenerator/python/SeedGeneratorFromProtoTracksEDProducer_cff.py]: @makortel 
Please you do that part after this PR as I am not the expert in this... Thanks!

Comment (makortel) [inline RecoTracker/TkSeedGenerator/python/SeedGeneratorFromProtoTracksEDProducer_cff.py]: Ok.

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/17438	1969bd5
- changes in HLT-related code; no impact on reco
- jenkins tests pass and comparisons with the baseline show no differences

**Comment by davidlange6** [comment]
+1

**Pull request merged**